grpc-client-rx-utils/build.gradle.kts

```diff
@@ -7,10 +7,16 @@ plugins {
 
 dependencies {
   api("io.reactivex.rxjava3:rxjava:3.0.6")
-  api("io.grpc:grpc-stub:1.40.0")
+  api("io.grpc:grpc-stub:1.42.0")
```

Comment:
I'm fine upgrading this, I always do, but we need to be really careful about upgrading since this is the kind of thing that leads to grpc version mismatches. That is, some service upgrades grpc utils, picks up grpc-stub, api, context of `1.42.0`, but is still using grpc-netty `1.39.0`

So we can broadcast it and watch for it on reviews, we could add gradle enforcement per repo, but wonder what the lightest weight way we can guarantee that not to happen is...

Reply:
I wonder if we could just add constraints on all of the grpc packages in here, like we're doing with gson? I'm pretty sure that would give the behavior we want.